From dc82ea5cdcc86b8c5574611076d1d5b467f5ae9f Mon Sep 17 00:00:00 2001
From: Herrie <github.com@herrie.org>
Date: Fri, 20 May 2016 20:28:04 +0200
Subject: [PATCH] Python3: Revert to Python 2 - Additional patch for VoiceCall
 version of oFono

Needed until we switch to Python 3. The VoiceCall version of oFono has
additional tests that need patching too.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 test/auto-answer-calls   | 2 +-
 test/get-network-time    | 2 +-
 test/test-voicecallagent | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/test/auto-answer-calls b/test/auto-answer-calls
index beea261..ade59dc 100644
--- a/test/auto-answer-calls
+++ b/test/auto-answer-calls
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
 
 from gi.repository import GLib
 
diff --git a/test/get-network-time b/test/get-network-time
index 2c2f40d..1dd7507 100644
--- a/test/get-network-time
+++ b/test/get-network-time
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
 
 import dbus
 import sys
diff --git a/test/test-voicecallagent b/test/test-voicecallagent
index ee48fec..86fee0e 100644
--- a/test/test-voicecallagent
+++ b/test/test-voicecallagent
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
 
 from gi.repository import GLib
 
